buildscripts/ci/translation/s3_packandsend.py

```diff
@@ -13,7 +13,7 @@
 #needs to be equal or smaller than the cron
 period = 300
 outputDir = "share/locale/"
-s3Urls = ["s3://extensions.musescore.org/4.3/languages/"]
+s3Urls = ["s3://extensions.musescore.org/4.4/languages/"]
```

Comment:
as a todo, it would be good to make reading from the config file (since it now exists :) )